config/mcus/SKR1.4_x2_V2.4.cfg

```diff
@@ -74,6 +74,9 @@ aliases:
 # recirculating filter
     FILTER_FAN=P2.7,
 
+# EXHAUST FAN
+    EXHAUST_FAN=P2.4,
```

Comment:
Can you change it to `P2.5` to avoid a duplicate pin (with z:CONTROLLER_FAN) ? Thanks :)